packages/solid-crs-presentation/lib/features/object/object.events.ts

```diff
@@ -23,17 +23,17 @@ export class SelectedObjectEvent implements EventObject {
 /**
  * Fired when the user clicks a sidebar item.
  */
-export class ClickedObjectSidebarItem implements EventObject {
+// export class ClickedObjectSidebarItem implements EventObject {
 
-  public type: ObjectEvents.CLICKED_SIDEBAR_ITEM = ObjectEvents.CLICKED_SIDEBAR_ITEM;
-  constructor(public itemId: string) { }
+//   public type: ObjectEvents.CLICKED_SIDEBAR_ITEM = ObjectEvents.CLICKED_SIDEBAR_ITEM;
+//   constructor(public itemId: string) { }
 
-}
+// }
 
 /**
  * Events for the object machine.
  */
 export type ObjectEvent =
 | SelectedObjectEvent
-| FormSubmittedEvent
-| ClickedObjectSidebarItem;
+// | ClickedObjectSidebarItem
+| FormSubmittedEvent;
```

Comment:
same here, there is no object sidebar in the presentation app, so can be removed